app/src/AppRouter.tsx

```diff
@@ -9,17 +9,20 @@ import AccessRequestPage from 'pages/access/AccessRequestPage';
 import LogInPage from 'pages/login/LogInPage';
 import LogOutPage from 'pages/logout/LogOutPage';
 import React from 'react';
-import { Redirect, Switch } from 'react-router-dom';
+import { Redirect, Switch, useLocation } from 'react-router-dom';
 import AppRoute from 'utils/AppRoute';
 import SearchPage from 'features/search/SearchPage';
 
 const AppRouter: React.FC = (props: any) => {
+  const location = useLocation();
+
   const getTitle = (page: string) => {
     return `BioHub - ${page}`;
   };
 
   return (
     <Switch>
+      <Redirect from="/:url*(/+)" to={location.pathname.slice(0, -1)} />
```

Comment:
what is this for?

Reply:
Oh oops, I thought I had updated the description (might have just been the commit message).

This is just some code that strips out trailing `/` in routes.
The routers have different behaviour if the url does or does not end in a slash, so this removes that concern.

Example:

You have a link that says `<Link to="newpage"/>`

The current URL in the browser is: `www.host.come/projects/`
That link will take you to: `www.host.come/currentpage/newpage`

The current URL in the browser is: `www.host.come/currentpage` (no trailing slash)
That link will take you to: `www.host.come/newpage`

Etc.